modules/cluster-logging-eventrouter-deploy.adoc

```diff
@@ -154,17 +154,17 @@ configmap/logging-eventrouter created
 deployment.apps/logging-eventrouter created
 ----
 
-. Validate that the Event Router installed:
+. Validate that the Event Router installed in the `openshift-logging` project:
```

Comment:
Will we drop Event router in 4.5?

Reply:
Yes. I just removed the two event router files. I removed event router from the TOC earlier.